apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerParameterTests.cs

```diff
@@ -46,6 +48,29 @@ public void DbTypeMappgings(SpannerDbType spannerType, DbType adoType)
             Assert.Equal(spannerType, parameter.SpannerDbType);
         }
 
+        // TODO: There is no value that will default to Spanner type DATE.
+        public static IEnumerable<object[]> GetValueConversions()
+        {
+            yield return new object[] { new byte[] { 1 }, SpannerDbType.Bytes, DbType.Binary };
+            yield return new object[] { true, SpannerDbType.Bool, DbType.Boolean };
+            yield return new object[] { new DateTime(2021, 1, 13, 12, 3, 10), SpannerDbType.Timestamp, DbType.DateTime };
+            yield return new object[] { 3.14D, SpannerDbType.Float64, DbType.Double };
```

Comment:
Can you add a case for float and decimal as well which currently map to Float64? Actually, for all of these, be exhaustive.

Reply:
I've added additional test cases for the currently existing mappings.